src/translation-assistant.js

```diff
@@ -458,8 +454,18 @@ function populateTemplate(template, englishStr, lang) {
         // string.
         const englishDictionary = getMathDictionary(
             englishStr, englishStr, lang);
-        const englishMapping = getMapping(
-            englishStr, englishStr, 'en', MATH_REGEX, englishDictionary);
+
+        let englishMapping;
+
+        // This call to getMapping() should be safe because we're
+        // comparing English string with itself. Nevertheless, we're cautious
+        // so we don't crash the Translation Editor.
+        try {
+            englishMapping = getMapping(
+                englishStr, englishStr, 'en', MATH_REGEX, englishDictionary);
+        } catch (error) {
+            return undefined;
```

Comment:
I wonder if we should log console.error here?

Reply:
Added.

Reply:
@mpolyak the linter doesn't like console.error. Should I disable it for this specific line? (not sure how, but I can find out)

Reply:
Maybe `// eslint-disable-next-line no-console`

Reply:
This didn't work, I guess because of outdated eslint. I used `eslint-disable-line no-console` instead.
I also added comments/examples that you requested.